lib/ansible/modules/cloud/vmware/vmware_host_powermgmt_policy.py

```diff
@@ -0,0 +1,210 @@
+            elif policy == 'custom':
+                policy_key = 4
+                policy_short_name = "custom"
+
+            power_system = host.configManager.powerSystem
+
+            # get current power policy
+            power_system_info = power_system.info
+            current_host_power_policy = power_system_info.currentPolicy
+            # current_host_power_policy.key
+            # current_host_power_policy.name
+            # current_host_power_policy.shortName
+            # current_host_power_policy.description
+            if current_host_power_policy.shortName == "static":
+                current_policy = 'high-performance'
+            elif current_host_power_policy.shortName == "dynamic":
+                current_policy = 'balanced'
+            elif current_host_power_policy.shortName == "low":
+                current_policy = 'low-power'
+            elif current_host_power_policy.shortName == "custom":
+                current_policy = 'custom'
+            if current_host_power_policy.key == policy_key:
+                results['result'][host.name]['msg'] = "power policy for host '%s' already configured" % host.name
+            else:
+                # get available power policies and check if policy is included
+                supported_policy = False
+                power_system_capability = power_system.capability
+                available_host_power_policies = power_system_capability.availablePolicy
+                for available_policy in available_host_power_policies:
+                    # available_policy.key
+                    # available_policy.name
+                    # available_policy.shortName
+                    # available_policy.description
+                    if available_policy.shortName == policy_short_name:
+                        supported_policy = True
+                if supported_policy:
+                    if not self.module.check_mode:
+                        try:
+                            power_system.ConfigurePowerPolicy(key=policy_key)
+                            changed = True
+                            results['result'][host.name]['msg'] = "Power policy for host '%s' changed from '%s' to '%s'" % \
+                                                                  (host.name, current_policy, policy)
+                        except vmodl.fault.InvalidArgument:
+                            self.module.fail_json(msg="Invalid power policy key for host '%s' provided" % host.name)
+                        except vim.fault.HostConfigFault as host_config_fault:
+                            self.module.fail_json(msg="Failed to configure power policy for host '%s': %s" %
+                                                  (host.name, to_native(host_config_fault.msg)))
+                    else:
+                        changed = True
+                        results['result'][host.name]['msg'] = "Power policy for host '%s' will be changed from '%s' to '%s'" % \
```

Comment:
Check mode should return the same output as an actual non-check mode. So it should return the same message as well. Ansible itselfs marks the output as 'check_mode', indicating a dry run

Reply:
Is this the Ansible "standard" to have similar output in all modes? 

I'm ok with that, but where is the output marked as check mode? The output would look similar and only the ansible-playbook command is different?